examples/component/wasm/guest.rs

```diff
@@ -0,0 +1,17 @@
+use wit_bindgen;
```

Comment:
I think this `use` can be omitted?

Also, can this file have a comment pointing to the `wit-bindgen` repository for more information about the generated bindings?

Reply:
Done.